tools/cleanupTarchives.pl

```diff
@@ -91,34 +129,38 @@
 ####     Main program     ####
 ##############################
 
-# Get tarchives list from the database and stores ArchiveLocation and md5sumArchive informations in a hash.
-## ArchiveLocation will be the key of the hash
-## md5sumArchive will be the value of the hash
-my ($tarchivesList_db)  = &selectTarchives($dbh, $tarchiveLibraryDir);
+# Get the list of DICOM archives the tarchive table of the database and stores
```

Comment:
__Gets__ the list of DICOM archives __from__ the tarchive table of the database and...

Reply:
done